### PR TITLE
PLF-6559 : Ability to disable the BloomFilter via property configurat…

### PR DESCRIPTION
**Pull request by aboughzela**
PLF-6559 : Ability to disable the BloomFilter via property configuration.

### CODE REVIEW

File: extension/config/src/main/resources/conf/platform/configuration.properties [hunk 1]
```diff
@@ -549,8 +549,16 @@ webdav.def-file-mimetype=${exo.webdav.def-file-mimetype:application/octet-stream
...
 webdav.auto-version=${exo.webdav.auto-version:checkin-checkout}
 webdav.folder-icon-path=${exo.webdav.folder-icon-path:/eXoWCMResources/skin/images/file/nt-folder.png}
-webdav.cache-control=${exo.webdav.cache-control:text/*:max-age=3600;image/*:max-age=1800;application/*:max-age=1800;*/*:no-cache} 
-
+webdav.cache-control=${exo.webdav.cache-control:text/*:max-age=3600;image/*:max-age=1800;application/*:max-age=1800;*/*:no-cache}
+
+#JCR Bloom Filter configuration
+gatein.jcr.bloomfilter.system.enabled=${exo.jcr.bloomfilter.system.enabled:true}
+gatein.jcr.bloomfilter.portal-system.enabled=${exo.jcr.bloomfilter.portal-system.enabled:true}
+gatein.jcr.storage.portal-work.enabled=${exo.jcr.storage.portal-work.enabled}:true
```
Comment (thomasdelhomenie) [inline extension/config/src/main/resources/conf/platform/configuration.properties]: Not related to bloom filter, should be removed IMO.

**Comment by thomasdelhomenie** [comment]
We must also update the file platform/extension/config/src/main/resources/conf/platform/exo-sample.properties with these new properties.

File: extension/config/src/main/resources/conf/platform/configuration.properties (hunk 1 above)
Comment (aboughzela) [inline extension/config/src/main/resources/conf/platform/configuration.properties]: property name fixed

**Comment by aboughzela** [comment]
exo-sample.properties updated.

**Pull request merged**